src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let res = sourceString.split(';');
+
+  res = res.map((el) => el.split(':'));
+  res = res.map((el) => el.map((item) => item.trim()));
+  res = res.filter((item) => item.length > 1);
```

Comment:
Filtering items with `item.length > 1` is a good approach to remove invalid entries. However, consider logging or handling these cases to inform the user about ignored entries.